project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module

```diff
@@ -475,7 +475,7 @@ function _c4m_document_widget_document_file($file_id, $is_quick_post = FALSE) {
   // PURL options.
   $options_context = array(
     'purl' => array(
-      'disabled' => FALSE,
+      'enabled' => TRUE,
```

Comment:
@anvmn This modification will cause an error in attaching files + adding to library. We should discuss.